libsrc/apple2/get_tv.s

```diff
@@ -0,0 +1,151 @@
+        .export     _get_tv
+
+        .import     _set_iigs_speed, _get_iigs_speed
+        .import     _bzero, _memset, pushax
+        .import     ostype
+
+        .importzp   tmp1, ptr1
+
+        .constructor calibrate_tv, 2
+
+        .include    "apple2.inc"
+        .include    "accelerator.inc"
+
+.segment "ONCE"
+
+; Cycle wasters
+waste_72:
+        jsr     waste_36
+waste_36:
+        jsr     waste_12
+waste_24:
+        jsr     waste_12
+waste_12:
+        rts
```

Comment:
As discussed, cycle wasting should include a reference to the paddle circuit in order to (likely) cause accelerators to slow down.

Reply:
Is there any other softswitch I can read that would slow the accelerators down? If I bit PTRIG in the loop, it resets the //c's VBL.